src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    value: 'Under construction',
+  },
```

Comment:
As part of the project requirements, kindly note that this should be an array of categories with `initial state` set to empty array `(initial state: empty array)` similar to what you did in the bookSlice and not a string as you have done here `(value: 'under construction')`. Please update this section to match the project requirements. :pray:

Reply:
Thank you so much for your review, I have made the required adjustments.